src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const tBody = document.querySelector('tbody');
+const minLimit = 2;
+const maxLimit = 10;
+const fields = {
+  rows: tBody.rows.length,
+  cols: tBody.rows[0].cells.length,
```

Comment:
Ensure that tBody.rows[0] exists before accessing tBody.rows[0].cells.length to avoid potential errors if the table body is empty.